book/forms.rst

```diff
@@ -77,8 +77,8 @@ from inside a controller::
     // src/AppBundle/Controller/DefaultController.php
     namespace AppBundle\Controller;
 
-    use Symfony\Bundle\FrameworkBundle\Controller\Controller;
     use AppBundle\Entity\Task;
+    use Symfony\Bundle\FrameworkBundle\Controller\Controller;
```

Comment:
From that I saw in others code I think most/all the time it's sorted alphabetically, just sometimes they are grouped adding the empty line between namespaces, but the groups are still sorted.

And when I use sublime editor it's not a lot of problems to do that, just add use statement anywhere, select a block and press F5, no need to process anything.